jha/src/components/ReviewTable.vue

```diff
@@ -5,7 +5,7 @@
         :key="element.label + '-' + index"
         class="bcgov-row d-flex">
       <div class="bcgov-table-cell label">
-        <div class="table-cell-max">{{element.label}}</div>
+        <div :class="'table-cell-max '  + isUnderlined(element)">{{element.label}}</div>
```

Comment:
This might be a good location to use Vue class binding rather than another function
https://v2.vuejs.org/v2/guide/class-and-style.html#Object-Syntax

Reply:
Something like:

`<div :class="['table-cell-max', { 'underlined': element.underlined }]">{{element.label}}</div>`

should work. `underlined` doesn't have to be wrapped in single quotes but I'd do it to make it easier to tell what the class is